examples/nextjs-with-typescript/pages/MediaChromeReactSimple.tsx

```diff
@@ -0,0 +1,396 @@
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import 'media-chrome';
+import styles from '../styles/Home.module.css';
+import {
+  Provider as MediaChromeProvider,
+  useMediaDispatch as useDispatch,
+  useFullscreenRefCallback,
+  useMediaRefCallback,
+  useMediaStore,
+  useMediaSelector as useSelector,
+} from '../components/media-chrome-react/stateMgmt';
+import 'media-chrome/dist/themes/microvideo.js';
+import { useEffect } from 'react';
+import { MediaUIEvents } from '../../../dist/constants';
+
+const PlayButton = () => {
+  const dispatch = useDispatch();
+  const mediaPaused = useSelector((state) => state.mediaPaused);
+  return (
+    <button
+      style={{ cursor: 'pointer' }}
+      onClick={() => {
+        const type = mediaPaused ? 'mediaplayrequest' : 'mediapauserequest';
```

Comment:
since these event names will be more public and frequently used just thought of building a new shelter type building for my bicycle here :) 

we good with the naming? the `request` part arguably could be left off, just a thought.
it could also be namespaced potentially `MediaRequestEvents.PLAY` or `MediaActions.PLAY`...

Reply:
tl;dr preamble - per our prior convo, let's make sure we keep any of the react-specific stuff out of scope for this, since none of it needs to be set in stone for our initial merge. If preferred, I can drop all of the examples here from the branch/PR.

Now, to your actual question:
I like this question/these ideas. I'd be down with a rename. Keep in mind that these are (at least currently) used by both the "actual" eventing system of media controller/Media Chrome web components and the media store (as a simple pass through), so if we rename that may introduce complexity or require a renaming of the events themselves. The constants lookup table may be a good resolution here and is fairly common in, e.g. the redux world for actions. @heff thoughts.

Reply:
A couple of worries.
* We bubble events a lot, out of the shadow dom, so I worry about some of our events leaking into other systems and causing problems if they have generic or overlapping names like `play`.
* These events are intended to separate user intentions from actual media state changes, so anything that makes them look more like the related media events I think will cause confusion.

I think I could get behind dropping the `media` part, but I still worry about conflicts down the line. Is `MediaUserEvents.PLAY_REQUEST` really better than `mediaplayrequest`?